impl/src/main/java/org/ehcache/config/builders/ConfigurationBuilder.java

```diff
@@ -122,4 +125,16 @@ public ConfigurationBuilder withClassLoader(ClassLoader classLoader) {
   public boolean containsCache(String alias) {
     return caches.containsKey(alias);
   }
+
+  public List<ServiceCreationConfiguration<?>> addDiscoveredServiceConfiguration() {
+    List<ServiceCreationConfiguration<?>> newServiceConfigurations = new ArrayList<>(serviceConfigurations);
```

Comment:
Why would the discovered ones be overridden by the explicit ones?

Reply:
because we want to have management activated by default in the EE kit but users could still programmatically or through xml change the management config to customize it.